pkg/webui/components/table/table/table.styl

```
@@ -28,6 +28,16 @@
 
 .section
 
+  // Hacky way of extending the header row by adding a divider between the header and the body.
+  &-divider
+    position: absolute
+    top: 42px
+    left: -20px
+    height: 1px
+    width: 98.5rem
+    background: var(--c-border-neutral-light)
+    z-index: 9999
```

```suggestion
  &-divider
    position: absolute
    bottom: 0
    left: - $cs.l
    right: - $cs.l
    height: 1px
    background: var(--c-border-neutral-light)
```

Don't make it more hacky than necessary :v: 

Also you need to remove the border bottom of the header row to avoid a double border.

Finally, please move this divider inside of the header row, so that it correctly "stickies" with the header row when scrolling.